Doc/library/asyncio-task.rst

```diff
@@ -1153,6 +1153,11 @@ Task Object
 
       Return the coroutine object wrapped by the :class:`Task`.
 
+      .. note::
```

Comment:
Add a versionchanged directive.

Reply:
Resolved in the new PR https://github.com/python/cpython/pull/104304.